backend/server/server.go

```diff
@@ -0,0 +1,78 @@
+package server
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/VenturaDelMonte/go-plate-checker/backend/platechecker"
+	"github.com/VenturaDelMonte/go-plate-checker/backend/store"
+	"github.com/google/uuid"
+	"github.com/indiependente/pkg/logger"
+	"github.com/labstack/echo"
+	"github.com/labstack/echo/middleware"
+)
+
+// PlateCheckerServer exposes endpoints to serve plate checking capabilities.
+type PlateCheckerServer struct {
+	e  *echo.Echo
+	pc platechecker.PlateChecker
+	s  store.PlateStore
+	l  logger.Logger
+}
+
+// NewPlateCheckerServer returns a new instance of a PlateCheckerServer.
+func NewPlateCheckerServer(pc platechecker.PlateChecker, s store.PlateStore, l logger.Logger) PlateCheckerServer {
+	e := echo.New()
+	e.Use(middleware.Logger())
+	e.Use(middleware.Gzip())
+	e.Use(middleware.RequestIDWithConfig(middleware.RequestIDConfig{
+		Generator: func() string {
+			return uuid.New().String()
+		},
+	}))
+	e.HTTPErrorHandler = customErrorHandler
+
+	return PlateCheckerServer{
```

Comment:
fose dovremmo usare delle variabili nella struct piu' chiare?

Reply:
Is this clear enough?
```
type PlateCheckerServer struct {
	e     *echo.Echo
	pc    platechecker.PlateChecker
	store store.PlateStore
	log   logger.Logger
}
```

Reply:
Much better!